src/com/lftechnology/realworldassign/ProductServices.java

```diff
@@ -0,0 +1,52 @@
+package com.lftechnology.realworldassign;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class consists of reusable methods of the Product items such as remove and update of the items in the List.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class ProductServices {
+	private static final Logger LOGGER = Logger.getLogger(ProductServices.class.getName());
+
+	/**
+	 * This method removes {@link String} value from the {@link List}
+	 * 
+	 * @param inputList
+	 *            is {@link List} from which value is to be removed
+	 * @param value
+	 *            is{@link String} to be removed
+	 */
+	public static void removeItem(List<String> inputList, String value) {
+		if (inputList.contains(value)) {
+			inputList.remove(value);
```

Comment:
Could have directly used `inputList.remove(value);` it returns false if there are is no element specified for remove in List.